container/ssl.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -ex
+
+mkdir -p /var/www/plugins/MAP /var/www/plugins/.ssl
+openssl req -x509 -newkey rsa:4096 -sha256 -nodes -keyout /var/www/plugins/.ssl/tmxcmd.key -out /var/www/plugins/.ssl/tmxcmd.crt -subj "/CN= <127.0.0.1> " -days 3650
```

Comment:
Is this script new? Does this allow for https access to the web UI or what is it's purpose.

Reply:
addressed